src/adapter/maker/SDaiOracle.sol

```diff
@@ -1,56 +1,33 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity 0.8.23;
 
-import {FixedPointMathLib} from "@solady/utils/FixedPointMathLib.sol";
-import {BaseAdapter, Errors} from "src/adapter/BaseAdapter.sol";
-import {IPot} from "src/adapter/maker/IPot.sol";
+import {IERC4626} from "forge-std/interfaces/IERC4626.sol";
+import {BaseAdapter, Errors, IPriceOracle} from "src/adapter/BaseAdapter.sol";
 
 /// @title SDaiOracle
 /// @author Euler Labs (https://www.eulerlabs.com/)
-/// @notice Adapter for pricing Maker sDAI <-> DAI via the DSR Pot contract.
+/// @notice Adapter for pricing Maker sDai <-> Dai.
 contract SDaiOracle is BaseAdapter {
```

Comment:
We were going to kill this adapter and rely on the router's `convertToAssets` right?